README.md

```diff
@@ -23,3 +23,33 @@ Todo REST API samples using ASP.NET Core minimal APIs. It showcases:
 1. Run `dotnet user-jwts create --claim id=myid`
 1. You should be able to use this JWT token to make requests to the endpoint
 1. Learn more about [user-jwts](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/minimal-apis/security?view=aspnetcore-7.0#using-dotnet-user-jwts-to-improve-development-time-testing)
+
+## Optional
+
+### OpenTelemetry
```

Comment:
Add instructions to uncomment the lines in the TodoApi/OpenTelemetryExtensions.cs file.